django/contrib/staticfiles/views.py

```diff
@@ -27,6 +29,12 @@ def serve(request, path, insecure=False, **kwargs):
 
     It uses the django.views.static.serve() view to serve the found files.
     """
+    warnings.warn(
+        "django.contrib.staticfiles.views.serve is deprecated",
```

Comment:
Upgrade instructions?

I think we would also want to deprecated `django.views.static.serve()`?